ddln_imaginator.proto

```diff
@@ -21,6 +21,64 @@ enum ImageFormat {
     GRAY = 4;
 }
 
+message Environment {
+    message Clouds {
+        enum Type {
+            NONE = 0;
```

Comment:
add comments describing the clouds
you can use a link to a sample image as a comment

Reply:
I'll link to this: https://drive.google.com/open?id=1uHQPM2e83kYv06pKwH-S1NGQRH_-I7CR

Reply:
I think the names are already sufficient. People can use google if they are unsure.

Reply:
If we have the simulation development driver, I'll drop all the info there.